tests/test_tpp_backend.py

```diff
@@ -6021,3 +6025,100 @@ def test_ons_cis():
         country_date=["2021-10-01", "2021-10-01", "2020-10-01"],
         country_as_codes=["0", "1", ""],
     )
+
+
+def test_ukrr():
+    "Test UK Renal Registry"
+    session = make_session()
+    session.add_all(
+        [
+            Patient(
+                UKRR=[
+                    UKRR(
+                        dataset="2019prev",
+                        renal_centre="AD100",
+                        rrt_start="2019-01-01",
+                        mod_start="RT",
+                        mod_prev="NULL",
```

Comment:
I'm not entirely confident about this, but I think you want None here, and sqlalchemy will make it a null in the db.  (Unless nulls in the real data are represented by the string "NULL".)